velum-bootstrap/spec/features/bootstrap_cluster.rb

```
@@ -13,6 +13,12 @@
     end
   end
 
+  # Using append after in place of after, as recommended by
+  # https://github.com/mattheworiordan/capybara-screenshot#common-problems
+  append_after(:each) do
```

good catch, jfyi I silently dropped `capybara-screenshot` again as it was not really useful. but this might even help in our scenario